srcpkgs/minikube/template

```diff
@@ -14,7 +14,8 @@ license="Apache-2.0"
 homepage="https://github.com/kubernetes/minikube"
 changelog="https://raw.githubusercontent.com/kubernetes/minikube/master/CHANGELOG.md"
 distfiles="https://github.com/kubernetes/minikube/archive/v$version.tar.gz"
-checksum="c3f0f60607bca0c2f1a118d6d5ce313ac0c6e56b5f9621cb00ca17271aceaff6"
+checksum=4ade34adf8d33ddd141d436b33cb66d4183e059f8a112ea1aeb1edeae9302594
+go_ldflags="-X k8s.io/minikube/pkg/version.version=$version -X k8s.io/minikube/pkg/version.isoVersion=$version"
```

Comment:
I think this flag goes below `go_import_path` for xlint to be happy.

Reply:
Yes, there's currently another issue I'd like to fix first before I care about xlint :wink:

Reply:
isoVersion seems to be ignored.